CDCgov/data-exchange-upload
# Upload 953/post finish hook

Adds the post finish hook to complete the dex-upload PS API stage.

Testing steps:
1. Log onto GFE
2. Navigate to `tus/file-hooks/post-create` directory
3. Create a trace by running the following command: `python post_create_bin.py --id "9876" --metadata "{\"meta_destination_id\": \"1234\", \"meta_ext_event\": \"test event\"}"
4. Query PS API at the `https://ocio-ede-dev-pstatus-api-azurewebsites.net/api/trace/uploadId/9876` and verify trace is returned with one child span with `stage_name` of "dex-upload" and `status` of "running".
5. Stop the child span by running the post finish hook with the following command: `python post_finish_bin.py --id "9876"`
6. Query the PS API again with the same request used in step 4 and verify the child span returned has a status of `completed`.
